src/icalendar/prop/__init__.py

```diff
@@ -911,6 +911,361 @@ def from_jcal(cls, jcal_property: list) -> Self:
+
+    def to_ical(self) -> bytes:
+        """Generate vCard format with semicolon-separated fields."""
+        parts = [vText(f).to_ical().decode(DEFAULT_ENCODING) for f in self.fields]
+        return ";".join(parts).encode(DEFAULT_ENCODING)
+
+    @staticmethod
+    def from_ical(ical: str | bytes) -> tuple[str, ...]:
+        """Parse vCard N format into tuple of 5 fields.
+
+        Args:
+            ical: vCard format string with semicolon-separated fields
+
+        Returns:
+            Tuple of 5 field values (empty string if field is empty)
+        """
+        from icalendar.parser import split_on_unescaped_semicolon
+
+        ical = to_unicode(ical)
+        fields = split_on_unescaped_semicolon(ical)
+        if len(fields) != 5:
+            raise ValueError(f"N must have exactly 5 fields, got {len(fields)}: {ical}")
+        return tuple(fields)
+
+    def __eq__(self, other):
+        """self == other"""
+        return isinstance(other, vN) and self.fields == other.fields
+
+    def __repr__(self):
+        """String representation."""
+        return f"{self.__class__.__name__}({self.fields}, params={self.params})"
+
+    from icalendar.param import VALUE
+
+    def to_jcal(self, name: str) -> list:
+        """The jCal representation of this property according to :rfc:`7265`."""
+        result = [name, self.params.to_jcal(), self.VALUE.lower()]
+        result.extend(self.fields)
+        return result
+
+    @classmethod
+    def from_jcal(cls, jcal_property: list) -> Self:
+        """Parse jCal from :rfc:`7265`.
+
+        Args:
+            jcal_property: The jCal property to parse.
+
+        Raises:
+            ~error.JCalParsingError: If the provided jCal is invalid.
+        """
+        JCalParsingError.validate_property(jcal_property, cls)
+        if len(jcal_property) != 8:  # name, params, value_type, 5 fields
+            raise JCalParsingError(
+                f"N must have 8 elements (name, params, value_type, 5 fields), "
+                f"got {len(jcal_property)}"
+            )
+        for i, field in enumerate(jcal_property[3:], start=3):
+            JCalParsingError.validate_value_type(field, str, cls, i)
+        return cls(
+            tuple(jcal_property[3:]),
+            Parameters.from_jcal_property(jcal_property),
+        )
+
+    @classmethod
+    def examples(cls) -> list[vN]:
+        """Examples of vN."""
+        return [cls(("Doe", "John", "M.", "Dr.", "Jr."))]
+
+
+class vOrg:
+    """vCard ORG (Organization) structured property per :rfc:`6350` Section 6.6.4.
+
+    The ORG property represents an organization with variable semicolon-separated fields:
+    organization-name, organizational-unit-1, organizational-unit-2, ...
+
+    At least one field (organization name) is required. Additional organizational units
+    are optional.
+
+    Semicolons are field separators and are NOT escaped.
+    Commas and backslashes within field values ARE escaped per :rfc:`6350`.
```

Comment:
I noticed that the description didn't much the example, so I double-checked the RFC. I'm not sure where you got the text because it's very different from the RFC. Can you double-check this?

```suggestion
    The ORG property specifies the organizational name and units associated with the vCard.
    
    Its value is a structured type consisting of components separated by semicolons.
    The components are the organization name, followed by zero or more levels of organizational unit names:

    organization-name; organizational-unit-1; organizational-unit-2; ...

    Semicolons are field separators and are NOT escaped.
    Commas and backslashes within field values ARE escaped per :rfc:`6350`.
```